File: assignment4/median.c
//March 3rd 2016
//Mamoutou Sangare V00010526
//This program takes three integer inputs;it prints them and the median number
#include<stdio.h>

int main(){
	
  int x, y,z, median;
  printf("Enter The first number: ");
  scanf("%d",&x);
  printf("Enter the second number: ");
  scanf("%d",&y);
  printf("Enter the third number: ");
  scanf("%d",&z);
  printf(" The three numbers entered: %d %d %d\n", x, y, z);
  printf("");
  median = findMedianVal(x,y,z);
  printf("The median:%d\n", median);	
  return 0;
}
int findMedianVal (int num1,int num2,int num3){

	  if ((num1 - num2) * (num3 - num1) >= 0) {
        return num1;
    } else if ((num2 - num1) * (num3 - num2) >= 0) {
        return num2;
    } else {
        return num3;
    }
}

File: assignment4/Median.java
//March 3rd 2016
//Mamoutou Sangare V00010526
//This program takes three integer inputs;it prints them and the median number
import java.util.Arrays;
import java.util.Scanner;

public class Median {
	
    private static Scanner scanner = new Scanner(System.in);
    
	public static void main(String[] args) {
		
		int number1, number2, number3;
		System.out.print("Enter the First Number: ");
		number1 = scanner.nextInt();
		System.out.print("Enter the Second Number: ");
		number2 = scanner.nextInt();
		System.out.print("Enter the Third Number: ");
		number3 = scanner.nextInt();
		System.out.println ("The three numbers entered: " + number1 + " " + number2 + " " + number3);
		int median = findMedianVal(number1,number2,number3);
		System.out.println("The median: " + median);	
	}
	
	private static int findMedianVal (int num1,int num2,int num3){
		int [] array = { num1, num2, num3};
		 Arrays.sort(array);
		 return array[1];   // second element in the array is the median
	}
}
